src/test/java/com/google/javascript/clutz/empty_type_args.d.ts

```diff
@@ -0,0 +1,31 @@
+declare namespace ಠ_ಠ.clutz_internal.empty_type_args {
+  interface ITemplated < T > {
+  }
+}
+declare module 'goog:empty_type_args.ITemplated' {
+  import alias = ಠ_ಠ.clutz_internal.empty_type_args.ITemplated;
+  export default alias;
+}
+declare namespace ಠ_ಠ.clutz_internal.empty_type_args {
+  class NoMoreTemplateArgs implements ITemplated < number > {
```

Comment:
Yeah, our golden masters are getting a bit wordy.

This is all just part of the buildup - the problem was that when `NoMoreTemplateArgs` was used, it was printed as `NoMoreTemplateArgs < >`, which is bad syntax in TS.